pkg/tools/etcd_helper.go

```diff
@@ -155,11 +164,14 @@ func (h *EtcdHelper) ExtractToList(key string, listObj runtime.Object) error {
 // a zero object of the requested type, or an error, depending on ignoreNotFound. Treats
 // empty responses and nil response nodes exactly like a not found error.
 func (h *EtcdHelper) ExtractObj(key string, objPtr runtime.Object, ignoreNotFound bool) error {
+	key = h.prefixEtcdKey(key)
```

Comment:
So here we prefix the key, and then we pass the prefixed key to h.bodyAndExtractObj and we pass the already prefixed key.  The first line in bodyAndExtractObj is to prefix the key again.  Doesn't this result in a key with the prefix twice?

This is why I asked if we had a convention in place for passing prefixed or non-prefixed keys across each function so readers can make sense of it.